app/src/main/java/com/kickstarter/models/Update.java

```diff
@@ -18,6 +18,7 @@
   public abstract @Nullable Integer commentsCount();
   public abstract @Nullable Boolean hasLiked();
   public abstract long id();
+  public abstract boolean isPublic();
```

Comment:
Is this setting the right value from the API, or is it always just initialized to `false`? Our serializer names this field `public` rather than `is_public`, and I couldn't see if we're handling that here.